tests/instrumentation/psycopg2_tests.py

```diff
@@ -507,3 +510,18 @@ def test_psycopg2_execute_values(instrument, postgres_connection, elasticapm_cli
         spans = elasticapm_client.spans_for_transaction(transactions[0])
         assert spans[0]["name"] == "INSERT INTO test"
         assert len(spans[0]["context"]["db"]["statement"]) == 10000, spans[0]["context"]["db"]["statement"]
+
+
+@pytest.mark.integrationtest
+def test_psycopg2_connection(instrument, elasticapm_transaction, postgres_connection):
+    elasticapm_client = cast(TempStoreClient, get_client())
```

Comment:
What is the purpose of this line? From the docs:

> Cast a value to a type.
>
> This returns the value unchanged. To the type checker this signals that the return value has the designated type, but at runtime we intentionally don’t check anything (we want this to be as fast as possible).

Seems like it doesn't do anything, and I can't see how the type checker cares here.

Reply:
a line down, we access `elasticapm_client.events`, which is only available on the `TempStoreClient`. Without it, PyCharm (and mypy, once we get around to activate that) will complain.

Reply:
Thanks! Can you add a comment here as well? Something like

```
# elastciapm_client.events is only available on `TempStoreClient`, this keeps the type checkers happy
```